css/css-pseudo/reference/selection-input-011-ref.html

```diff
@@ -19,6 +19,33 @@
     }
   </style>
 
+  <script>
+  function startReference()
+  {
+  document.getElementById("ref").focus();
+  /*
```

Comment:
We usually put comments above the code which is explained by the comment.

Reply:
Done in 961685d